src/components/simple-color-picker/simple-color-picker.spec.tsx

```diff
@@ -71,19 +81,23 @@ describe("SimpleColorPicker", () => {
     ));
   });
 
-  describe("it renders children in rows based on maxWidth and childWith", () => {
-    let wrapper, onChange, secondColor;
+  describe("it renders children in rows based on maxWidth and childWidth", () => {
+    let wrapper: ReactWrapper;
+    let onChange: jest.Mock;
+    let secondColor: ReactWrapper<
+      SimpleColorProps & React.RefAttributes<HTMLInputElement>
+    >;
 
     describe("onKeyDown", () => {
       beforeEach(() => {
         onChange = jest.fn();
-        wrapper = render({ maxWidth: "58", childWith: "58", onChange });
+        wrapper = render({ maxWidth: "58", childWidth: "58", onChange });
       });
       it("fires onKeyDown callback if provided", () => {
         const onKeyDown = jest.fn();
         wrapper = render({
           maxWidth: "58",
-          childWith: "58",
+          childWidth: "58",
```

Comment:
I'm curious how these tests even passed before with the typo in the prop name! I only mention this because it hints to me that the test is not actually very useful.

Reply:
Yeah, the reason why it passed before is that the `SimpleColorPicker` has a default value for `childWidth` equal to `58` set already.

Reply:
I've changed these values to be slightly different now